CorsixTH/Lua/dialogs/information.lua

```diff
@@ -57,10 +57,10 @@ function UIInformation:UIInformation(ui, text, use_built_in_font)
   -- Window size parameters
   self.text_width = 300
   self.spacing = {
-    l = 40,
-    r = 40,
-    t = 20,
-    b = 20,
+    l = 15,
+    r = 15,
+    t = 15,
+    b = 33, -- includes space for close button (18px + 15px padding)
```

Comment:
```suggestion
    b = 18 + 15, -- Size of  close button + padding
```
Not sure if the comment is fully correct, but in this way you let the computer add the numbers, so comment and reality always match.

Reply:
Happy to go with that for now